sklearn/metrics/_scorer.py

```diff
@@ -498,10 +498,17 @@ def make_scorer(score_func, *, greater_is_better=True, needs_proba=False,
                 needs_threshold=False, **kwargs):
     """Make a scorer from a performance metric or loss function.
 
-    This factory function wraps scoring functions for use in GridSearchCV
-    and cross_val_score. It takes a score function, such as ``accuracy_score``,
-    ``mean_squared_error``, ``adjusted_rand_index`` or ``average_precision``
+    This factory function wraps scoring functions for use in
+    :class:`~sklearn.model_selection.GridSearchCV` and
+    :func:`~sklearn.model_selection.cross_val_score`.
+    It takes a score function, such as :func:`~sklearn.metrics.accuracy_score`,
+    :func:`~sklearn.metrics.mean_squared_error`,
+    :func:`~sklearn.metrics.adjusted_rand_index` or
+    :func:`~sklearn.metrics.average_precision`
     and returns a callable that scores an estimator's output.
+    The signature of the call is ``(estimator, X, y)`` where ``estimator``
+    is the model to be evaluated, ``X`` is the test data and ``y`` is the
+    ground truth labeling (or ``None`` in the case of unsupervised models).
```

Comment:
Using "test data" here may be confusing because the scorer can run on any data (including training data).

Nit: Moving toward single backticks for new code because its nicer to read in a code editor?

```suggestion
    The signature of the call is `(estimator, X, y)` where `estimator`
    is the model to be evaluated, `X` is the test data and `y` is the
    ground truth labeling (or `None` in the case of unsupervised models).
```

Reply:
No problem. I copied it from the docstring at the start of the `_scoroer.py` file actually.